### PR TITLE
Update acm and github action version

### PR DESCRIPTION
**Pull request by theprashantyadav**
## what
* update acm and aws version
* fix tflint and update github action version 
* remove unuse argument in main.tf
## why
* need to update aws version
* need to update github-action version

### CODE REVIEW

**Review by themaniskshah — DISMISSED** [review]
LGTM!

File: _example/nlb/exampe.tf [hunk 1]
```diff
@@ -103,7 +103,7 @@ module "ec2" {
 
 module "acm" {
```
Comment (nileshgadgi) [inline _example/nlb/exampe.tf:104]: please correct the file name.

**Review by pendharevaibhav-cd — APPROVED** [review]
LGTM!

**Review by nileshgadgi — APPROVED** [review]
LGTM!

**Review by anmolnagpal — APPROVED** [review]

**Pull request merged**